actions/release_homebrew/action.yml

```diff
@@ -173,10 +176,41 @@ runs:
         directory: ${{ github.workspace }}/release_homebrew_action/homebrew_core_fork_repo
         force: true  # need to force since the branch is sometimes reset
         github_token: ${{ inputs.token }}
-        message: "Update ${{ github.repository }} to ${{ github.sha }}"
+        message: "${{ github.repository }}: update to ${{ github.sha }}"
         repository: ${{ inputs.homebrew_core_fork_repo }}
 
-    - name: Create Pull Request
+    - name: Create Pull Request (org homebrew repo)
+      env:
+        GH_TOKEN: ${{ inputs.token }}
+        INPUT_ORG_HOMEBREW_REPO_BRANCH: ${{ inputs.org_homebrew_repo_branch }}
+        INPUT_ORG_HOMEBREW_REPO: ${{ inputs.org_homebrew_repo }}
+      if: ${{ inputs.publish == 'true' }}
+      shell: bash
+      working-directory: ${{ github.workspace }}/release_homebrew_action/org_homebrew_repo
+      run: |
+        # Check if a pull request already exists with the same head branch
+        PR_EXISTS=$(gh pr list \
+          --head ${INPUT_ORG_HOMEBREW_REPO_BRANCH} \
+          --repo ${INPUT_ORG_HOMEBREW_REPO})
+
+        # If the pull request does not exist, create it
+        if [[ -z "$PR_EXISTS" ]]; then
+          echo "Creating pull request"
+
+          # https://cli.github.com/manual/gh_pr_create
+          gh pr create \
+            --base master \
+            --head ${INPUT_ORG_HOMEBREW_REPO_BRANCH} \
+            --title "chore: update ${{ github.repository }} to ${{ github.sha }}" \
+            --body \
+              "Created by the LizardByte [release_homebrew](https://github.com/LizardByte/actions) action" \
+            --no-maintainer-edit \
```

Comment:
```suggestion
```

homebrew core repo also needs this removed, as they modify PRs to add bottles

also, automerge the PR in our org repo: https://cli.github.com/manual/gh_pr_merge